libs/fluxcd/config.jsonnet

```diff
@@ -87,5 +87,47 @@ config.new(
       ],
       localName: 'fluxcd',
     },
+    {
```

Comment:
It's a bit hard to extract a version table in here, probably not worth the effort without prior automation and update management.